lexer_test.go

```diff
@@ -23,8 +23,10 @@ func TestLexer(t *testing.T) {
 		DISTINCT ALL JOIN ON USING EXISTS IS NOT FILTER
 		CREATE TABLE INT BLOB ANY PRIMARY KEY UNIQUE CHECK DEFAULT GENERATED ALWAYS STORED VIRTUAL CONSTRAINT
 		INSERT INTO VALUES DELETE UPDATE SET
-		GRANT TO REVOKE
-	`
+		GRANT TO REVOKE 
+		"doublequoteidentifier" [squarebracketsidentifier]
```

Comment:
Added 3 examples to our test. 

This is how this test works. It is just a long string with strings separated by space inside. I test if those strings emit the correct token.